README.md

```diff
@@ -120,6 +120,7 @@ You can also use premailer from the command line by using his main module.
     --disable-basic-attributes DISABLE_BASIC_ATTRIBUTES
                           Disable provided basic attributes (comma separated)
     --disable-validation  Disable CSSParser validation of attributes and values
+		--pretty              Pretty-print the outputted HTML.
```

Comment:
There's some strange indentation going on there. Both the `--pretty` and the `Pretty-print the...` are over-indented.